content/applications/websites.rst

```diff
@@ -9,4 +9,5 @@ Websites
 
     websites/website
     websites/ecommerce
+    websites/forum
```

Comment:
Hi @AntoineVDV 

could you add the team "forum-doc-review" as code owner of this new folder, please? Thanks!

Reply:
done